### PR TITLE
fix: logging tab error about modules

### PR DESCRIPTION
**Pull request by chmelevskij**
Fixes that console error about import only being usable in modules.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by limonspb — APPROVED** [review]

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
